eurocropsml/dataset/preprocess.py

```diff
@@ -153,7 +153,7 @@ def get_class_ids_to_names(raw_data_dir: Path) -> dict[str, str]:
 
 
 def _find_padding(array: np.ndarray) -> bool:
-    if np.array_equal(array, np.array([0] * len(array))):
+    if np.array_equal(array, np.array([-999] * len(array))):
```

Comment:
@jmaces: This wouldn't work with the current Sentinel-2 raw data. In the past, some values returned zero for all bands during the clipping process. I could adjust the Sentinel-2 data to replace the zeros with `None`, which would cause them to be filtered out in this module. However, in this case I think I should give a warning when downloading older data that it does not work with this version of the package.

Reply:
You mean with older versions of the Zenodo data then?
Sounds like a good plan to make it work with the most recent versions, and warn for other incompatible versions :+1:

Reply:
Yes, exactly. 
Alright, I'll add it to the other PR :)